<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Writer</class>
 <widget class="QWidget" name="Writer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="2" column="1">
      <widget class="QLineEdit" name="toLine"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>To</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="fromLine"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>From</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="titleLine"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="copyAllButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="edit-copy"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="document-save"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveAsButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="document-save-as"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="undoButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="edit-undo"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="redoButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="edit-redo"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Font Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fontSizecomboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Font:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fontcomboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="editor"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
